mediawiki:latest/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM centos:latest
+MAINTAINER Vessel Maintainers
```

Comment:
Who is this?

Reply:
Ah, I saw that. Meant to change it. Very early work on ASB concept.